src/target.zig

```diff
@@ -11,7 +11,7 @@ pub const ArchOsAbi = struct {
 };
 
 pub const available_libcs = [_]ArchOsAbi{
-    .{ .arch = .aarch64_be, .os = .linux, .abi = .gnu },
+    .{ .arch = .aarch64_be, .os = .linux, .abi = .gnu }, // glibc aarch64 v2.17+
```

Comment:
instead of putting this data in a comment, how about putting it into the data? then you can use it in glibc.zig instead of duplicating the data.

Reply:
Adding the versions here (as an optional `.glibc_min` field) means the existing `canBuildLibC` call in target.zig can do the GNU library version checking too.  And that means the glibc.zig doesn't need to do any version bounds checks, which really simplified things.